### PR TITLE
task: refactor harvesters script to support Symbol in addition to NEM

### PR DESCRIPTION
**Pull request by Jaguar0625**

### CODE REVIEW

File: lint.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
+set -ex
+
 find . -type f -name "*.sh" -print0 | xargs -0 shellcheck
-find . -type f -name "*.py" -print0 | PYTHONPATH=. xargs -0 "$(pyenv which isort)" --check-only --line-length 140
-find . -type f -name "*.py" -print0 | PYTHONPATH=. xargs -0 "$(pyenv which pycodestyle)" --config=.pycodestyle
-find . -type f -name "*.py" -print0 | PYTHONPATH=. xargs -0 "$(pyenv which pylint)" --load-plugins pylint_quotes
+find . -type f -name "*.py" -print0 | PYTHONPATH=. xargs -0 "$(command -v isort)" --check-only --line-length 140
```
Comment (gimre-xymcity) [inline lint.sh:6]: `command`? 😅  what's that?

File: templates/symbol.mainnet.yaml [hunk 2]
```diff
@@ -10,8 +9,7 @@ accounts:
...
 nodes:
-  - host: ngl-dual-001.symbolblockchain.io
-  - host: ngl-dual-002.symbolblockchain.io
-  - host: ngl-dual-003.symbolblockchain.io
-  - host: ngl-dual-004.symbolblockchain.io
-  - host: ngl-dual-005.symbolblockchain.io
+  - host: wolf.importance.jp
+  - host: 00fabf14.xym.stir-hosyu.com
+  - host: symbol.shizuilab.com
+  - host: symbol.ooo
```
Comment (gimre-xymcity) [inline templates/symbol.mainnet.yaml:15]: what are those hardcoded hosts? 😁

File: lint.sh (hunk 1 above)
Comment (Jaguar0625) [reply lint.sh:6]: `command -v` is apparently more widely distributed `which` (shellcheck complained about which)

File: templates/symbol.mainnet.yaml (hunk 2 above)
Comment (Jaguar0625) [reply templates/symbol.mainnet.yaml:15]: just random ones i picked from list

**Review by gimre-xymcity — APPROVED** [review]

**Pull request merged**